spk/ffsync/src/requirements-all.txt

```diff
@@ -0,0 +1,21 @@
+# From requirements.txt
+
+# We pin cryptography to 3.4.8 due to a build error that occurs with the
+# current version (36.0.0 at the time of this commit). In short, building
+# the cryptography package with Rust results in a segfault, so we need to
+# set the CRYPTOGRAPHY_DONT_BUILD_RUST env var to force the package to build
+# with C instead. This env var is only present in cryptography<3.5.
+cryptography==3.4.8
+pyfxa==0.7.7
+tokenlib==2.0.0
+
+# From tools/tokenserver/requirements.txt
+
+boto==2.49.0
+hawkauthlib==2.0.0
+mysqlclient==2.1.1
+pyramid==2.0.1
+sqlalchemy==1.4.46
+testfixtures
+tokenlib==2.0.0
+PyBrowserID==0.14.0
```

Comment:
You don't need a requirements-all.txt
Only need the `crossenv` and `pure`, sometimes abi3 but probably not applicable in here.
The "merged" `requirements.txt` used later will get auto-populated at build time.

Reply:
I can delete this file, I just left it for reference (for now) of what I separated manually. I don't think it should cause an issue as it's not referenced in the Makefile.